help_to_heat/frontdoor/interface.py

```diff
@@ -544,6 +544,10 @@ def get_epc_details(self, lmk):
         epc_api = EPCApi()
         return epc_api.get_epc_details(lmk)
 
+    def get_epc_recommendations(self, lmk):
+        epc_api = EPCApi()
+        return epc_api.get_epc_recommendations(lmk)
```

Comment:
Feels like this would be the appropriate level to combine all the data we get back from the API (unless there's a reason not to?) - would merge this with the above method and return an object with both `epc_details` and `recommendations`.

Reply:
have merged them